packages/amplify-graphql-api-construct-tests/src/__tests__/sql-iam-access.test.ts

```diff
@@ -0,0 +1,337 @@
+import * as path from 'path';
```

Comment:
This file is result of splitting the `sql-models.test.ts`.

This file is result of moving "new iam access tests" outside of `sql-models.test.ts`.